assembl/static2/css/views/synthesis.scss

```diff
@@ -132,3 +132,8 @@
     height: 2000px;
   }
 }
+
+.button-container {
+  display: flex;
+  justify-content: space-around;
+}
```

Comment:
We use the `button-container` class in other places. We need to very there is no regression elsewhere.